sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -599,9 +599,6 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
         if (item.partitionKey) {
           entry["partition-key"] = item.partitionKey;
         }
-        if (item.viaPartitionKey) {
-          entry["via-partition-key"] = item.viaPartitionKey;
-        }
```

Comment:
Since we will be adding this back in the near future, I wonder if it is better to comment the code out rather than delete it. @HarshaNalluru, @richardpark-msft Thoughts?

Reply:
Agreed, better to keep them commented.

Reply:
@mohsin-mehmood This comment is applicable to all other places from where we are deleting the code for viaPartitionKey. Can you comment such code instead of deleting it?

Also, I see the below comment you added

> // Will be required later for implementing [Transactions feature of Service Bus](https://docs.microsoft.com/azure/service-bus-messaging/service-bus-transactions).

This uses the markdown syntax for links which we dont need in source code. You can instead use

> // Will be required later for implementing Transactions